MapboxCoreNavigation/EventDetails.swift

```diff
@@ -365,42 +365,18 @@ extension UIApplicationState: Encodable {
 
 extension AVAudioSession {
     var audioType: String {
-        if isOutputBluetooth() {
+        if currentRoute.outputs.contains(where: { [.bluetoothA2DP, .bluetoothHFP, .bluetoothLE].contains($0.portType) }) {
```

Comment:
This could just be a simple switch statement inside a for loop, but it looks like the original code was prioritizing Bluetooth types over headphone types over speaker types.